### PR TITLE
[libvirt] enable vencrypt when libvirt tls enabled

### PR DESCRIPTION
**Pull request by stuggi**
Depends-On: https://github.com/openstack-k8s-operators/openstack-operator/pull/780

Jira: [OSPRH-6174](https://issues.redhat.com//browse/OSPRH-6174)

### CODE REVIEW

File: roles/edpm_libvirt/templates/qemu.conf.j2 [hunk 1]
```diff
@@ -1,7 +1,12 @@
 max_files = 32768
 max_processes = 131072
+{% if edpm_libvirt_tls_certs_enabled|bool %}
+vnc_tls = 1
+vnc_tls_x509_verify = 1
+{% else -%}
 vnc_tls = 0
 vnc_tls_x509_verify = 0
```
Comment (olliewalsh) [inline roles/edpm_libvirt/templates/qemu.conf.j2:8]: can omit `vnc_tls_x509_verify`, that's already the default unless we set `default_tls_x509_verify`

Comment (olliewalsh) [reply roles/edpm_libvirt/templates/qemu.conf.j2:8]: also isn't relevant when vnc_tls = 0

**Comment by stuggi** [comment]
recheck

**Review by gibizer — APPROVED** [review]

**Pull request merged**